backend/routes/courses.js

```diff
@@ -298,4 +298,45 @@ router.post("/updateCourse", async (req, res, next) => {
   }
 });
 
+/**
+ *
+ * @api {post} /assignCourseToCurriculum
+ * @apiDescription assign a course to a curriculum (core, techComp, complementaries)
+ * @apiParam (body) {string} curriculumType, {string} courseCode, {string} curriculum
```

Comment:
Can you specify what values the curriculumType param should take? since the type has to have exact values to match

Reply:
Actually I put it in line 304, gonna update curriculum to curriculumType

Reply:
Ah ok i see, thats good then!